### PR TITLE
ref(ui): Alert List

### PR DESCRIPTION
**Pull request by robinrendle**
- Added tooltip to Alert Badges 
- Fixed titles for Alert Wizard (this is the last time, ugh)
- Tidied up Breadcrumbs for Wizard
- Fixed the column with the project link

### CODE REVIEW

File: static/app/components/idBadge/projectBadge.tsx [hunk 1]
```diff
@@ -59,6 +59,7 @@ const ProjectBadge = ({
 
 const StyledLink = styled(Link)`
   flex-shrink: 0;
+  width: 100%;
```
Comment (robinrendle) [inline static/app/components/idBadge/projectBadge.tsx:62]: Fixes the column issue with the ProjectLink. Might cause issues elsewhere, so I need to double check the snapshots.

Comment (scttcper) [reply static/app/components/idBadge/projectBadge.tsx:62]: i fixed in https://github.com/getsentry/sentry/pull/25506

Comment (scttcper) [reply static/app/components/idBadge/projectBadge.tsx:62]:
```suggestion
```

File: static/app/views/alerts/rules/row.tsx [hunk 2]
```diff
@@ -344,6 +361,7 @@ const AlertName = styled('div')`
 
 const ProjectBadge = styled(IdBadge)`
   flex-shrink: 0;
+  width: 100% !important;
```
Comment (scttcper) [inline static/app/views/alerts/rules/row.tsx:364]:
```suggestion
```

**Review by scttcper — APPROVED** [review]

**Pull request merged**